app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/ruling_details.scala.html

```diff
@@ -55,7 +53,7 @@ <h3 class="heading-medium  mt-1">@messages("case.decision.application.title")</h
     <div class="grid-row">
         <div class="column-one-third">
             <h3 class="heading-medium mt-0">@messages("case.menu.ruling-details")</h3>
-            @if(c.status == CaseStatus.OPEN && request.hasWritePermission) {
+            @if(c.status == CaseStatus.OPEN && request.hasPermission(Permission.EDIT_RULING) ) {
```

Comment:
Just a thought: we could potentially encompass the `CaseStatus.OPEN` validation within the permission adding/removing logic too. Thoughts?

Reply:
could be a good idea as long as the request contains the case

Reply:
Not sure I follow this.  What has a user having permission to edit (any) ruling got to do with the ability to edit the ruling on a particular case if it's not open?

Reply:
Basically if the case isnt open, you dont have permission to edit the ruling @SteveSugden

Reply:
So you're suggesting that we filter (remove) the users permissions based on the state of the case?   So for each CaseStatus we could (potentially) have a set of Permissions that are not valid and thus remove them for the user.   Nice idea but I'm wondering if it would add an extra layer of complexity by mixing concerns.